include/application/pokegear/pokegear_internal.h

```diff
@@ -152,8 +154,8 @@ BOOL PokegearApp_HandleInputModeChangeToButtons(PokegearAppData *pokegearApp);
 int PokegearApp_HandleTouchInput_SwitchApps(PokegearAppData *pokegearApp);
 int PokegearApp_HandleKeyInput_SwitchApps(PokegearAppData *pokegearApp);
 BOOL ov100_021E5BB0(PokegearAppData *pokegearApp, BOOL a1);
-void ov100_021E5CA4(PokegearAppData *pokegearApp);
-BOOL ov100_021E5D3C(PokegearAppData *pokegearApp, int a1);
+void Pokegear_ClearAppBgLayers(PokegearAppData *pokegearApp);
+BOOL Pokegear_RunFadeLyrs123(PokegearAppData *pokegearApp, int a1);
```

Comment:
I don't think there's much to be gained by saving 2 characters on this name
```suggestion
BOOL Pokegear_RunFadeLayers123(PokegearAppData *pokegearApp, int a1);
```